diff --git a/vsrc/small/nvdla_small.v b/vsrc/small/nvdla_small.v
index 69c4904..129db5d 100644
--- a/vsrc/small/nvdla_small.v
+++ b/vsrc/small/nvdla_small.v
@@ -84,8 +84,8 @@ NV_NVDLA_apb2csb apb2csb (
 NV_nvdla nvdla_top (
    .dla_core_clk                    (core_clk)
   ,.dla_csb_clk                     (csb_clk)
-  ,.global_clk_ovr_on               (1'b0)
-  ,.tmc2slcg_disable_clock_gating   (1'b0)
+  ,.global_clk_ovr_on               (1'b1)
+  ,.tmc2slcg_disable_clock_gating   (1'b1)
   ,.dla_reset_rstn                  (rstn)
   ,.direct_reset_                   (1'b1)
   ,.test_mode                       (1'b0)
